### ISSUE TITLE
"The required attribute regionName was not specified"

### ISSUE THREAD

**Issue by macintacos** [mongodb/terraform-provider-mongodbatlas#34]
I received the following error:

```
Error: error creating MongoDB Cluster: POST https://cloud.mongodb.com/api/atlas/v1.0/groups/redacted/clusters: 400 (request "Bad Request") The required attribute regionName was not specified.
```

Using the following code:

```
locals {
  public_key  = "redacted"
  private_key = "redacted"
  project_id  = "redacted"
}

variable "cluster_name" {
  description = "The name to use for the cluster you'd like to create"
  type        = string
}

provider "mongodbatlas" {
  public_key  = local.public_key
  private_key = local.private_key
}

data "mongodbatlas_clusters" "all_clusters" {
  project_id = local.project_id
}

resource "mongodbatlas_cluster" "creating_cluster" {
  project_id                  = local.project_id
  name                        = var.cluster_name
  provider_name               = "AWS"
  provider_instance_size_name = "M10"
}
```

This error was only thrown when I used `terraform apply`; `terraform plan` did not throw this error. Also, I was following along with this doc [here](https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/website/docs/r/cluster.html.markdown), and it didn't mention anything about the region being a required parameter to be set. Easy enough to address, just wanted to call it out.

**Comment by marinsalinas** [comment]
I think when is a single region cluster you should add this value (@themantissa  can you confirm this?): 
`provider_region_name   ="US_EAST_1"`

Could you try adding this value.

**Comment by macintacos** [comment]
@marinsalinas yep! It works fine when I specify the other parameters that come up when I try `terraform apply`, so the issue I have is that:

- `terraform plan` didn't catch this (I'm new to Terraform in general so I don't know if it was supposed to catch it or not)
- the documentation that I linked does not note that this is a required parameter.

**Comment by marinsalinas** [comment]
Yes, we have to specify when we try to create a single region cluster, we must add this value, let me add this to the documentation.

Thanks!

**Comment by PacoDw** [comment]
We'll move this issue to github.com/terraform-providers/terraform-provoder-mongodbatlas

**Comment by themantissa** [comment]
@PacoDw if you move these over to finish please resolve here once moved.
